ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/RedHat.yml

```diff
@@ -172,7 +181,10 @@
 - name: Install Java when RedHat 8 and above
   package: "name={{ item }} state=latest"
   with_items: "{{ Java_RHEL8Plus }}"
-  when: (ansible_distribution_major_version | int >= 8)
+  when:
+    - (ansible_distribution_major_version | int >= 8)
```

Comment:
May be simpler to read if the conditional was "==8 OR ==9", especially since one has `>=` and the other `<` at the moment.

Although I'm now curious as to what this isn't applicable for RHEL6/7 ...

Reply:
Ive fixed this, but I've no idea why it isn't applicable to rhel6 & 7.